db/fixtures/customization_templates.yml

```diff
@@ -274,3 +274,10 @@
     />\n</unattend>"
   :type: CustomizationTemplateSysprep
   :system: true
+
+- :name: Basic root pass template
+  :description: This template takes use of rootpassword defined in the UI
+  :script: "#cloud-config\nchpasswd:\n  list: |\n    root:<%= MiqPassword.decrypt(evm[:root_password]) %>\n  expire: False"
+  :type: CustomizationTemplateCloudInit
+  :pxe_image_type_id: 1
```

Comment:
You should not specify the `:pxe_image_type_id` value as the target database will likely not have a matching instance.  If the appliance is running with a region other than `REGION 0` it definitely will not match.